docs/userguide/datafiles.rst

```diff
@@ -35,16 +37,34 @@ For example, if the package tree looks like this::
             ├── data1.txt
             └── data2.txt
 
-and you supply this configuration:
+When at least one of the following conditions are met:
```

Comment:
The `include_package_data` keyword would only have an effect when condition is met, perhaps it's good to state these upfront?